tests/filter/bare.c

```diff
@@ -0,0 +1,114 @@
+#include "clar_libgit2.h"
+#include "crlf.h"
+
+static git_repository *g_repo = NULL;
+static git_blob_filter_options filter_opts;
+
+void test_filter_bare__initialize(void)
+{
+	cl_fixture_sandbox("crlf.git");
+	cl_git_pass(git_repository_open(&g_repo, "crlf.git"));
+
+	filter_opts.flags |= GIT_BLOB_FILTER_NO_SYSTEM_ATTRIBUTES;
+	filter_opts.flags |= GIT_BLOB_FILTER_ATTTRIBUTES_FROM_HEAD;
```

Comment:
If we were correctly checking the version of the struct then we'd produce an error, right? So first, we need to initialize `filter_opts` correctly so that its version is set to "1". And second, we need to include the options version check

Reply:
Fixed.